### ISSUE TITLE
Notebook is executed with errors.

### ISSUE THREAD

**Issue by giasemidis** [mljar/mercury#179]
My notebooks used to run fine, now they give this error:

deployed app [here](https://espn-nba-fantasy.herokuapp.com/)

```
Notebook is executed with errors.
Traceback (most recent call last): File "/app/.heroku/python/bin/jupyter-nbconvert", line 8, in <module> sys.exit(main()) File "/app/.heroku/python/lib/python3.9/site-packages/jupyter_core/application.py", line 269, in launch_instance return super().launch_instance(argv=argv, **kwargs) File "/app/.heroku/python/lib/python3.9/site-packages/traitlets/config/application.py", line 982, in launch_instance app.start() File "/app/.heroku/python/lib/python3.9/site-packages/nbconvert/nbconvertapp.py", line 361, in start self.convert_notebooks() File "/app/.heroku/python/lib/python3.9/site-packages/nbconvert/nbconvertapp.py", line 533, in convert_notebooks self.convert_single_notebook(notebook_filename) File "/app/.heroku/python/lib/python3.9/site-packages/nbconvert/nbconvertapp.py", line 498, in convert_single_notebook output, resources = self.export_single_notebook(notebook_filename, resources, input_buffer=input_buffer) File "/app/.heroku/python/lib/python3.9/site-packages/nbconvert/nbconvertapp.py", line 427, in export_single_notebook output, resources = self.exporter.from_filename(notebook_filename, resources=resources) File "/app/.heroku/python/lib/python3.9/site-packages/nbconvert/exporters/exporter.py", line 186, in from_filename return self.from_file(f, resources=resources, **kw) File "/app/.heroku/python/lib/python3.9/site-packages/nbconvert/exporters/exporter.py", line 204, in from_file return self.from_notebook_node(nbformat.read(file_stream, as_version=4), resources=resources, **kw) File "/app/.heroku/python/lib/python3.9/site-packages/nbconvert/exporters/html.py", line 145, in from_notebook_node return super().from_notebook_node(nb, resources, **kw) File "/app/.heroku/python/lib/python3.9/site-packages/nbconvert/exporters/templateexporter.py", line 392, in from_notebook_node output = self.template.render(nb=nb_copy, resources=resources) File "/app/.heroku/python/lib/python3.9/site-packages/jinja2/environment.py", line 1301, in render self.environment.handle_exception() File "/app/.heroku/python/lib/python3.9/site-packages/jinja2/environment.py", line 936, in handle_exception raise rewrite_traceback_stack(source=source) File "/app/.heroku/python/share/jupyter/nbconvert/templates/lab/index.html.j2", line 3, in top-level template code {% from 'jupyter_widgets.html.j2' import jupyter_widgets %} File "/app/.heroku/python/share/jupyter/nbconvert/templates/lab/base.html.j2", line 234, in top-level template code {% set div_id = uuid4() %} File "/app/.heroku/python/share/jupyter/nbconvert/templates/base/display_priority.j2", line 1, in top-level template code {%- extends 'base/null.j2' -%} File "/app/.heroku/python/share/jupyter/nbconvert/templates/base/null.j2", line 24, in top-level template code {%- block header -%} File "/app/.heroku/python/share/jupyter/nbconvert/templates/lab/index.html.j2", line 9, in block 'header' {%- block html_head -%} File "/app/.heroku/python/share/jupyter/nbconvert/templates/lab/index.html.j2", line 36, in block 'html_head' {% block notebook_css %} File "/app/.heroku/python/share/jupyter/nbconvert/templates/lab/index.html.j2", line 37, in block 'notebook_css' {{ resources.include_css("static/index.css") }} File "/app/.heroku/python/lib/python3.9/site-packages/nbconvert/exporters/html.py", line 151, in resources_include_css return jinja2.Markup(code) AttributeError: module 'jinja2' has no attribute 'Markup'
```

Similar error when I run the app locally.

**Comment by giasemidis** [comment]
I managed to fix it locally by downgrading to jinja2-3.0.1 from Jinja2-3.1.2.

**Comment by giasemidis** [comment]
I managed to fix it locally by downgrading to jinja2-3.0.1 from Jinja2-3.1.2

**Comment by giasemidis** [comment]
Error went away after I fixed the version of Jinja in my requirements file and redeployed.
